.gitignore

```diff
@@ -10,15 +10,17 @@ desktop.ini
 .DS_Store
 ._*
 
-# Travis
-/build/phpdoc-*/
-/build/phpdoc-*.git/
-
 # Composer
 /composer.lock
 /composer.phar
 /vendor
 
+# phpDocumentor
+/_build/phpdoc/
```

Comment:
I proposed the prefix `_`change not to avoid collision with the shell scripts, but rather to signify that the directory is not apart of Pico's main files. Hope that clears it up @dav-m85 :smile: